frontend/src/components/derive-options/derive-options.css

```
@@ -0,0 +1,470 @@
+  --d-muted: var(--muted-foreground);
+  --d-text: var(--foreground);
+  --d-buy: var(--green);
+  --d-sell: var(--red);
+  --d-call: var(--green);
+  --d-put: var(--red);
+  --d-itm: color-mix(in srgb, var(--green) 6%, transparent);
+  --d-chip: color-mix(in oklch, var(--muted) 40%, var(--background));
+  --d-chip-active: color-mix(in oklch, var(--muted) 70%, var(--background));
+  --d-accent: var(--primary);
+  --d-selected-fg: oklch(0.145 0 0);
+  background: var(--d-bg);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)